src/game/behaviors/door.inc.c

```diff
@@ -83,49 +90,55 @@ void bhv_door_loop(void) {
             play_warp_door_open_noise();
             break;
     }
-    if (o->oAction == DOOR_ACT_CLOSED) {
-        load_object_collision_model();
-    }
+
     bhv_door_rendering_loop();
 }
 
 void bhv_door_init(void) {
+    const f32 checkDist = 200.0f;
```

Comment:
Not a blocker since this is an unimportant function to optimize, but do you know if `const` gets optimized to the same thing as if you opted to use a macro here? Mostly asking for my own sake.